app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx

```diff
@@ -161,6 +162,46 @@ const PageRequestReview = ({
               </ChangeApprovalOperations>
             )}
 
+            <h5>Recommended actions before launch</h5>
```

Comment:
This should actually be an `h3`. We probably need to tweak some of these since I see some are `h2`s when "Review and Launch" is an `h2`, could be a low priority maintenance item.

Also! We use `mb-3` on all other headers on this page.

Reply:
That's not too big? In the mockups it looked like it should be smaller than the other titles?

Reply:
If the font needs to be smaller it can be done with classes, semantically speaking an `h5` should only occur if it's under an `h4` on the page!